src/Mailchimp/Synchronisation/RequestBuilder.php

```diff
@@ -391,11 +392,33 @@ private function getAdherentActiveTags(Adherent $adherent): array
             $tags[] = ReferentTagRepository::FRENCH_OUTSIDE_FRANCE_TAG;
         }
 
+        if ($adherent->isCertified()) {
+            $tags[] = MailchimpSegmentTagEnum::CERTIFIED;
+        }
+
+        if ($adherent->hasVotingCommitteeMembership()) {
+            $tags[] = MailchimpSegmentTagEnum::COMMITTEE_VOTER;
+        }
+
         return $tags;
     }
 
     private function getInactiveTags(Adherent $adherent): array
     {
-        return PostAddress::FRANCE === $adherent->getCountry() ? [ReferentTagRepository::FRENCH_OUTSIDE_FRANCE_TAG] : [];
+        $tags = [];
+
+        if (PostAddress::FRANCE === $adherent->getCountry()) {
+            $tags[] = ReferentTagRepository::FRENCH_OUTSIDE_FRANCE_TAG;
+        }
+
+        if (!$adherent->isCertified()) {
```

Comment:
It seems to be :
```suggestion
        if ($adherent->isCertified()) {
```

Reply:
no, here we are in `getInactiveTags`, so if the adherent is not certified, we need disable this tag